src/views/month.php

```diff
@@ -15,6 +15,9 @@
 
 do_action( 'tribe_events_before_template' );
 
+// Title Bar
+tribe_get_template_part( 'month/titlebar' );
```

Comment:
Lets use `month/title-bar` for the name of this file.

Reply:
Fixed.